src/libraries/BeforeTargetFrameworkInference.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
```

Comment:
Should this be in eng or are we OK with it in root of libraries?

Reply:
We are not using this anywhere else. so i think we are fine with having in the libraries root

Reply:
Our guidance was to put all/most msbuild files under /eng. As this is an official SDK hook point I think it makes sense to move it into eng.